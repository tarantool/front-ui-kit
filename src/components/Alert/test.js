import React from 'react';
import renderer from 'react-test-renderer';

import { Alert } from '.';

it('error type renders correctly', () => {
  const tree = renderer.create(<Alert type="error">Error alert</Alert>).toJSON();

  expect(tree).toMatchSnapshot();
});

it('success type renders correctly', () => {
  const tree = renderer.create(<Alert type="success">Success alert</Alert>).toJSON();

  expect(tree).toMatchSnapshot();
});
