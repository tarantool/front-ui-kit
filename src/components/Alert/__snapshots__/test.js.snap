// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error type renders correctly 1`] = `
.emotion-0 {
  margin: 0;
  font-family: Inter,Arial,sans-serif;
  font-size: 14px;
  line-height: 22px;
  color: #000000;
  -webkit-font-smoothing: antialiased;
  padding: 15px;
  border: 1px solid;
  border-radius: 4px;
  margin-top: 16px;
  margin-bottom: 16px;
  border-color: #fea39e;
  background-color: #fff1f0;
  color: #ff272c;
}

.emotion-0 b {
  font-weight: 600;
}

<div
  className="emotion-0"
>
  Error alert
</div>
`;

exports[`success type renders correctly 1`] = `
.emotion-0 {
  margin: 0;
  font-family: Inter,Arial,sans-serif;
  font-size: 14px;
  line-height: 22px;
  color: #000000;
  -webkit-font-smoothing: antialiased;
  padding: 15px;
  border: 1px solid;
  border-radius: 4px;
  margin-top: 16px;
  margin-bottom: 16px;
  border-color: #b5ec8e;
  background-color: #f6ffee;
  color: #52c41a;
}

.emotion-0 b {
  font-weight: 600;
}

<div
  className="emotion-0"
>
  Success alert
</div>
`;
